jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/util/JdrZipFile.java

```diff
@@ -135,8 +135,15 @@ public void add(VirtualFile file, InputStream is) throws Exception {
      * @throws Exception
      */
     public void add(String content, String path) throws Exception {
-        String name = "sos_strings/as7/" + path;
-        this.add(new ByteArrayInputStream(content.getBytes()), name);
+        StringBuilder name = new StringBuilder("sos_strings/");
+
+        name.append(this.env.getProductName().replace(" ", "_").toLowerCase());
+        name.append("-");
+        name.append(this.env.getProductVersion().split("\\.")[0]);
```

Comment:
This and L140 can NPE since JDRRunner is ignoring failures obtaining this information.